WordPress/Classes/Utility/App Configuration/AppConstants.swift

```diff
@@ -21,6 +21,13 @@ import WordPressAuthenticator
     #endif
 }
 
+// MARK: - Tab bar order
+@objc enum WPTab: Int {
+    case mySites
+    case notifications
+    case reader
```

Comment:
@leandroalonso I think this needs to be
```suggestion
    case reader
    case notifications
```

Reply:
Oops, someone ended up committing some test code.